lib/matplotlib/axes/_base.py

```diff
@@ -854,36 +854,50 @@ def _update_transScale(self):
                     pass
 
     def get_position(self, original=False):
-        'Return the a copy of the axes rectangle as a Bbox'
+        """
+        Get a copy of the axes rectangle as a `.Bbox`.
+
+        Parameters
+        ----------
+        original : bool
+            If ``True``, return the original position. Otherwise return the
+            active position. For an explanation of the positions see
+            `.set_position`.
+
+        Returns
+        -------
+        pos : `.Bbox`
+
+        """
         if original:
             return self._originalPosition.frozen()
         else:
             return self._position.frozen()
 
     def set_position(self, pos, which='both'):
-        """Set the axes position
+        """
+        Set the axes position.
 
-        The expected shape of ``pos`` is::
+        There are two position variables:
```

Comment:
This was the part I found a bit confusing, and I understand what you are talking about.  Maybe:

Axes have two position attributes: the 'original' position is the position allocated for the axes; the 'active' position is the position the Axes is actually drawn at. These positions will usually be the same except if the Axes has its aspect ratio set by `.apply_aspect`.